src/component.ts

```diff
@@ -64,5 +64,10 @@ export function componentFactory (
   const Super = superProto instanceof Vue
     ? superProto.constructor as VueClass
     : Vue
-  return Super.extend(options)
+  const rv = Super.extend(options);
+
+  for(let sttc in Component)
+    if(!(sttc in function() {}) && Component.hasOwnProperty(sttc))
```

Comment:
I don't know if `hasOwnProperty` will impact component inheritance. A test case should be helpful.

Reply:
Written like that, a child class doesn't forward the statics of its parent. `hasOwnProperty` means the the property is in the object, not its prototype.

I am not sure of the question, therefore I don't know if it answers - what do you mean by "impact component inheritance" ?

Reply:
Consider
```ts
@Compoent class Parent extends Vue { static config = {} }
@Compoent class Child extends Parent {}

Child.config // expect an object {}
```

> Written like that, a child class doesn't forward the statics of its parent. 

I think you already understand me. And I even suspect whether we should support inheritance any way.

Reply:
Indeed, `Child.config` would, in this implementation, retrieve `undefined`
Beside beeing the easiest to implement, I just checked on how typescript managed static' legacy, and it seems coherent with the usual behaviour.

Reply:
Let's keep it simple. vue-class-component is for both Babel and TS users. Rolling out a babel compatible version might be challenging.

Reply:
So, is it still a "change requested" or an acceptation?